### ISSUE TITLE
`chattr -<attributes> <target>`  not working

### ISSUE THREAD

**Issue by peterlobster** [twpayne/chezmoi#398]
Hello, I'm a new greenhorn trying to understand how to implement Chezmoi. 

I accidentally added the `private` attribute to my `.bashrc` file and now I can't remove it. When I try to remove ANY of the attributes (by prefixing the minus to the attribute), I get back an error. 

As I said, I'm pretty new to Chezmoi; so I'm wondering if perhaps I've misunderstood something and I'm doing something wrong?

```
$chezmoi chattr -private .bashrc
chezmoi: unknown shorthand flag: 'p' in -private
$chezmoi chattr -p .bashrc
chezmoi: unknown shorthand flag: 'p' in -p
```

**Comment by twpayne** [comment]
The `-` on its own causes chezmoi to interpret `private` as a flag. Use one of these forms:

```
chezmoi chattr -- -private ~/.bashrc
chezmoi chattr noprivate ~/.bashrc
```

**Comment by peterlobster** [comment]
@twpayne Thanks for the clarification. I tried both of those and they obviously worked.

However, I did try `chezmoi chattr noprivate ~/.bashrc` yesterday and I got this outputted. 

Maybe I was doing something wrong (like I didn't declare the `~/`, which might be the problem); the important thing is that it seems to be working now.

```bash
$chezmoi chattr noprivate .bashrc
chezmoi: : unknown attribute
```

Thanks again!
